Convex_hull_3/include/CGAL/convex_hull_3.h

```diff
@@ -624,8 +689,43 @@ void non_coplanar_quickhull_3(std::list<typename Traits::Point_3>& points,
 //  CGAL_ch_postcondition(is_strongly_convex_3(P, traits));
 }
 
+template <class Point_3, class OutputIterator>
+void add_isolated_points(std::list<Point_3>& points, Output_iterator_wrapper<OutputIterator>& w)
+{
+  BOOST_FOREACH(Point_3 p, points)
+    ++w.out = p;
+}
 
+template <class Point_3, class Polyhedron_3>
+void add_isolated_points(std::list<Point_3>& points, Polyhedron_3& P)
+{
+  typedef typename boost::graph_traits<Polyhedron_3>::vertex_descriptor vertex_descriptor;
+  typedef typename boost::property_map<Polyhedron_3, CGAL::vertex_point_t>::type Vpmap;
+  Vpmap vpm = get(CGAL::vertex_point, P);
 
+  BOOST_FOREACH(Point_3 p, points)
+  {
+    vertex_descriptor v = add_vertex(P);
+    put(vpm, v, p);
+  }
+}
+
+template <class Point_3, class OutputIterator>
+void
+ch_triangle_3(Point_3& p1, Point_3& p2, Point_3& p3,
```

Comment:
no, add an overload for make_triangle close to the one added for make_tetrahedron